envoy/server/options.h

```diff
@@ -90,6 +90,12 @@ class Options {
    */
   virtual bool skipHotRestartOnNoParent() const PURE;
 
+  /**
+   * @return bool don't get stats from the parent. If there are a lot of stats, getting them
```

Comment:
nit: you don't need to specify the type in a doxygen return declaration.

Reply:
Not changed, for consistency with the doxygen comments nearby which say `@return bool`.